interface/src/index.html

```diff
@@ -93,9 +93,9 @@ <h1><a ng-href="/#/home" style="color:black">
         <div class="footer-inner row">
           <p class="text-center text-muted">
             <small>
-              tribe-interface v<%= version %> &copy; 2017 
-              <a href="http://www.greenelab.com/">Greene Laboratory</a>; 
-              <a href="http://www.med.upenn.edu/">Perelman School of Medicine</a> 
+              tribe-interface v<%= version %> &copy; 2017
```

Comment:
Can you also set a new version number and change `2017` in this line to `2018`? Thanks.